data/entities.yaml

```diff
@@ -185,6 +185,22 @@
   - It also possess some electro-magnetic properties.
   properties: [portable]
 
+- name: handle
```

Comment:
Actually `handle` already exists: #1028

Reply:
Created #1221 to catch this

Reply:
Whoops, thanks!  I'll fix this soon.

Reply:
Can confirm that once I merged #1223 into this branch it did catch the duplicate `handle`.  I changed `handle` to `keyboard`.